### PR TITLE
Android Back Up Input blurring

### PR DESCRIPTION
**Pull request by perunt**
Fixes RNBW-3405

## What changed (plus any additional context for devs)
You can change the display size in several gradations in the phone settings. If you set the display size more than the default, the text in the app will also increase, and the proportions will change.
I use the same solution as for small screens by hiding the icon from the screen

## PoW (screenshots / screen recordings)
before:
 - [pixel 5](https://streamable.com/nromee)
 - [pixel 4](https://streamable.com/am566f)
 after:
 - [pixel 4](https://streamable.com/7hftq2)
## Final checklist
Try to backup a wallet
- [x] Assigned individual reviewers?
- [x] Added labels?
- [x] Added e2e tests? if not please specify why
- [x] If you added new files, did you update the CODEOWNERS file?

### CODE REVIEW

**Review by osdnk — APPROVED** [review]

**Review by tchayen — CHANGES_REQUESTED** [review]
I can't reproduce the problem 😬.

I have one suggestion about the code.

File: src/components/backup/RestoreCloudStep.js [hunk 1]
```diff
@@ -78,6 +79,7 @@ export default function RestoreCloudStep({
   fromSettings,
 }) {
   const dispatch = useDispatch();
+  const { scale } = useDimensions();
   const { isTinyPhone } = useDimensions();
```
Comment (terrysahaidak) [inline src/components/backup/RestoreCloudStep.js:83]:
```suggestion
  const { isTinyPhone, scale } = useDimensions();
```

**Comment by perunt** [comment]
@tchayen yes, you're right, it doesn't matter here. 
You can see that on Android. Just set one of the largest display sizes in phone settings

**Review by terrysahaidak — APPROVED** [review]

**Review by tchayen — APPROVED** [review]
I still can't reproduce the problem on my testing phone, but also it's not breaking anything so 👌.

**Review by jkadamczyk — APPROVED** [review]

**Pull request merged**